### PR TITLE
added v2 to executions/exeId

### PR DESCRIPTION
**Pull request by ETtestim**

### CODE REVIEW

File: api.yaml [hunk 1]
```diff
@@ -678,6 +678,121 @@ paths:
...
+                        type: string
+                        description: The name of the execution
+                        example: remote-run
+                      concurrency:
+                        type: number
+                        description: The amount of tests executed in parallel
+                        example: 1
+                      executionResult:
+                        type: string
+                        description: Wether the execution passed or failed
```
Comment (liorama) [inline api.yaml]:
```suggestion
                        description: Whether the execution passed or failed
```

File: api.yaml [hunk 2]
```diff
@@ -678,6 +678,121 @@ paths:
...
+                        type: number
+                        description: The amount of failed tests
+                        example: 1
+                      totalTests:
+                        type: number
+                        description: The amount of tests executed
+                        example: 1
+                      link:
+                        type: string
+                        description: A link to the execution result in Testim editor
```
Comment (liorama) [inline api.yaml]:
```suggestion
                        description: A link to the execution result in Testim's editor
```

File: api.yaml [hunk 3]
```diff
@@ -678,6 +678,121 @@ paths:
...
+                      turboMode:
+                        type: boolean
+                        description: Wether the execution used Testim's Turbo Mode or not
```
Comment (liorama) [inline api.yaml]:
```suggestion
                        description: Whether the execution used Turbo Mode or not
```

File: api.yaml [hunk 4]
```diff
@@ -678,6 +678,121 @@ paths:
...
+                      duration:
+                        type: number
+                        description: The amount of time (MS) the execution took
+                      tests:
+                        type: array
+                        description: A list of the failed tests
```
Comment (liorama) [inline api.yaml]: These aren't the failed tests, right?
```suggestion
                        description: A list of all included tests
```

File: api.yaml [hunk 5]
```diff
@@ -678,6 +678,121 @@ paths:
             $ref: '#/components/responses/ExecutionDoesNotExist'
           '429':
             $ref: '#/components/responses/TooManyRequests'
+    /v2/runs/executions/{executionId}:
+      get:
+        summary: Get execution results
+        description: Get the result of an execution, together with its tests, for failed tests a list of the failed steps
```
Comment (liorama) [inline api.yaml]: I think it was decided to only return one list which has all test results, of all statuses.

File: api.yaml (hunk 4 above)
Comment (ETtestim) [reply api.yaml]: fixed

File: api.yaml (hunk 1 above)
Comment (ETtestim) [reply api.yaml]: fixed

File: api.yaml (hunk 2 above)
Comment (ETtestim) [reply api.yaml]: fixed

File: api.yaml (hunk 3 above)
Comment (ETtestim) [reply api.yaml]: fixed

File: api.yaml (hunk 5 above)
Comment (ETtestim) [reply api.yaml]: that is what i return but Ran said that for failed tests we should return the failed steps. I will check with him again.

Comment (ETtestim) [reply api.yaml]: spoke with Ran he asked to that we keep the same format for failed tests.
@liorama

**Review by barspielberg — APPROVED** [review]

**Pull request merged**